web/tests/test_scan.py

```
@@ -1,4 +1,5 @@
 import json
+import logging
```

Ruff

`logging` imported but unused

Remove unused import: `logging`

(F401)